src/leveldb/db/c.cc

```diff
@@ -5,7 +5,9 @@
 #include "leveldb/c.h"
```

Comment:
Changes to leveldb should go through upstream, either https://github.com/google/leveldb or if it's a specific change for us https://github.com/bitcoin-core/leveldb